webDemo/src/App.tsx

```diff
@@ -55,10 +55,14 @@ const itemsToRender: ItemToRender[] = [
           size={Button.sizes.large}
           iconSource={svgData}
           iconStyle={{
-            width: size,
-            height: size,
+            // width: size,
+            // height: size,
```

Comment:
If the `iconStyle` object isn't relevant anymore let's remove it.
Delete the commented lines.